scripts-available/CDB_Quota.sql

```diff
@@ -20,33 +20,50 @@ RETURNS bigint AS
 $$
 DECLARE
   total_size INT8;
+  raster_available BOOLEAN;
+  raster_read_query TEXT;
 BEGIN
+  -- Postgis 3+ might not install raster
```

Comment:
We have to account for Postgis3 not including raster face to migrations. Although never officially supported, I wonder if there's someone depending on raster yet. cc/ @manmorjim

Reply:
There were some cases that appeared during the last migration but, as of today, the idea is to drop postgis_raster and postgis_topology during the next migration, but we'll have to get into more detail about how.